hazelcast-client/src/main/java/com/hazelcast/client/cache/impl/AbstractClientCacheProxy.java

```diff
@@ -260,8 +260,8 @@ public V get(K key, ExpiryPolicy expiryPolicy) {
         Data expiryPolicyData = toData(expiryPolicy);
         ClientMessage request = CacheGetAllCodec.encodeRequest(nameWithPrefix, keySet, expiryPolicyData);
         ClientMessage responseMessage = invoke(request);
-        Set<Map.Entry<Data, Data>> entrySet = CacheGetAllCodec.decodeResponse(responseMessage).entrySet;
-        for (Map.Entry<Data, Data> dataEntry : entrySet) {
+        List<Map.Entry<Data, Data>> entries = CacheGetAllCodec.decodeResponse(responseMessage).entries;
+        for (Map.Entry<Data, Data> dataEntry : entries) {
```

Comment:
why do you call the collection entries, but a single item 'dataEntry'. So either uses entries/entry or dataEntries/dataEntry.